### ISSUE TITLE
DPUB tip and notice are not necessary for accessibility; use complementary or HTML aside

### ISSUE THREAD

**Issue by cookiecrook** [w3c/aria#57]
DPUB tip and notice are not necessary for accessibility as these are just use examples of `role="complementary"` or HTML `<aside>` whose additional context is conveyed via a heading or should be otherwise by an accessible graphic indicator.

**Comment by cookiecrook** [comment]
I could be persuaded otherwise on these two with some examples. The O'Reilly-style tips and notices seem like they can be conveyed clearly enough as is.

**Comment by Joanmarie** [comment]
Clearly I should read everything first, _then_ respond. :) As I just mentioned in issue #51, I think comments (like those which can be inserted in document editors) are role-worthy for accessibility and independent of DPUB.

Also see what @mattgarrish stated in issue #43. There are potentially different navigation / user-interaction expectations between tip and notice.

**Comment by TzviyaSiegman** [comment]
Thank you for your feedback. This issue has been resolved by editing the document to comply with the rules for extending ARIA (https://www.w3.org/WAI/PF/wiki/ARIAExtensions).
